tests/suites/utils.js

```diff
@@ -96,7 +96,7 @@ if (utils.gteVersion(phantom.version, '1.9.0')) {
                           modifiers.ctrl | modifiers.alt | modifiers.shift,
                          'computeModifier() computes a "ctrl+alt+shift" modifier');
         test.assertThrows(utils.computeModifier, ["chucknorris", modifiers],
-                         'computeModifier() checks for a valid modifier');
+                         'computeModifier() checks for a valid modifier '+ JSON.stringify(modifiers,null,4));
```

Comment:
This patch definitely does to match, please keep it focused on the feature you're actually trying to implement.